Dockerfile.oraclelinux7

```diff
@@ -161,3 +164,38 @@ RUN --mount=type=bind,source=.,target=rondb-src \
 # run with --output <output-folder>
 FROM scratch AS get-package-all
 COPY --from=build-all /root/rondb-tarball .
+
+##################################
+# Intermediate Image for RDRS ####
+##################################
+
+FROM ubuntu:22.04 AS rdrs-intermediate
+
+WORKDIR /root
+COPY --from=build-all /root/rondb-tarball .
```

Comment:
You don't need to copy the tarball - you can just copy from the bin directory? Unless the bin directory is for some reason incomplete.

i.e. use `--mount=type=cache,target=rondb-bin,id=oracle7-rondb2104-bin`

Then you also don't need two separate stages.

Reply:
I already tried it but I ran into problems relating to versioning of lib files. Plus rdrs is not in the bin folder of `oracle7-rondb2104-bin`. It is much cleaner to rely on the tarball than on cache.